lib/eventboss/error_handlers/db_connection_drop_handler.rb

```diff
@@ -0,0 +1,12 @@
+module Eventboss
+  module ErrorHandlers
+    class DbConnectionDropHandler
+      def call(exception, context = {})
+        if exception.class == ActiveRecord::StatementInvalid
```

Comment:
What if `eventboss` is used outside of Rails? I believe referring to `ActiveRecord` would result in _uninitialized constant_ exception 🤔

Reply:
Good point @ddknapik, but maybe it may be organised in for of middlewares like Rails or sidekiq? Then you can control which middlewares are used or not with some decent list of enables by default?

Reply:
This one is specifically for Rails (perhaps we can come up some more relevant name...).
If it's used outside of Rails we simply don't add it to `error_handlers` config.

Reply:
Well we should automatically add it to handlers if it's within Rails container, similar to
https://github.com/AirHelp/eventboss/blob/78a381e234618c4ff26c05c61f262c03c09fc14a/lib/eventboss/instrumentation.rb#L3

Reply:
Something like 
```ruby
handlers.add(xxx) if defined? ActiveRecord::Sth
```